tests/e2e/system_test/test_mcg_replication_with_disruptions.py

```diff
@@ -0,0 +1,248 @@
+import json
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import tier2
+
+from ocs_ci.ocs.node import get_worker_nodes, get_node_objs, wait_for_nodes_status
+from ocs_ci.ocs.bucket_utils import (
+    compare_bucket_object_list,
+    write_random_test_objects_to_bucket,
+    s3_delete_object,
+)
+
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.resources.pod import (
+    get_pod_objs,
+    delete_pods,
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.retry import retry
+from ocs_ci.framework.testlib import skipif_ocs_version
+from ocs_ci.ocs.exceptions import CommandFailed, ResourceWrongStatusException
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.9")
+class TestMCGReplicationWithDisruptions(E2ETest):
+
+    """
+    The objectives of this test case are:
+    1) To verify that namespaced buckets can be replicated across MCG clusters
+    2) To verify that the user can change from unidirectional MCG bucket replication to bidirectional successfully
+    3) To verify that the Data restore functionality works
+    4) To verify that the Certain admin/disruptive operations do not impact the replication
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["source_bucketclass", "target_bucketclass"],
+        argvalues=[
+            pytest.param(
+                {
+                    "interface": "oc",
+                    "namespace_policy_dict": {
+                        "type": "Multi",
+                        "namespacestore_dict": {
+                            "aws": [(1, "eu-central-1")],
+                            "azure": [(1, None)],
+                        },
+                    },
+                },
+                {
+                    "interface": "oc",
+                    "namespace_policy_dict": {
+                        "type": "Single",
+                        "namespacestore_dict": {"rgw": [(1, None)]},
+                    },
+                },
+                marks=[tier2, pytest.mark.polarion_id("OCS-2682")],
```

Comment:
Is this automated from the system_tests list?

Reply:
@PrasadDesala I need some clarity on polarion-id... this ID was added by mistake